fairscale/nn/data_parallel/fully_sharded_data_parallel.py

```diff
@@ -1816,13 +1821,11 @@ def wrap_bn_only_policy(module: nn.Module, recurse: bool, unwrapped_params: int)
                 module, tuple(default_auto_wrap_policy.EXCLUDE_WRAP_MODULES)  # type: ignore
             )
 
-    pg = None
+    pg = process_group
```

Comment:
why do weassign this if it is overwritten in the following lines?

Reply:
I prefer the code to be more concise. Instead of assign to None and then use if/else two cases, default to the parameter and then check if we need to create one when caller requests it is both clearer and faster to me.